tests/testthat/test-incidence.R

```diff
@@ -225,23 +225,23 @@ test_that("corner cases", {
                "The interval 'grind' is not valid. Please supply an integer.")
 
   expect_error(incidence(as.Date(Sys.Date()), last_date = "core"),
-               "last_date could not be converted to Date")
+               "last_date is a character. Did you forget to convert to Date?")
 
   expect_error(incidence(1, "week"),
                "The interval 'week' can only be used for Dates")
 
   expect_error(incidence(as.Date(Sys.Date()), standard = "TRUE"),
                "The argument `standard` must be either `TRUE` or `FALSE`")
-  
+
   expect_error(incidence(sample(10), intrval = 2),
```

Comment:
`intrval` should be `interval`; it will be silently ignored here due to `...`

Reply:
'intrval' was actually not a typo; it was a test to check the behavior for mis-spelled argument names; have left as-is